Swiftfin tvOS/Views/SearchView.swift

```diff
@@ -58,15 +60,35 @@ struct SearchView: View {
                 if viewModel.people.isNotEmpty {
                     itemsSection(title: L10n.people, keyPath: \.people, posterType: .portrait)
                 }
+
+                if viewModel.programs.isNotEmpty {
+                    itemsSection(title: L10n.programs, keyPath: \.programs, posterType: .landscape)
+                }
+
+                if viewModel.channels.isNotEmpty {
+                    itemsSection(title: L10n.channels, keyPath: \.channels, posterType: .portrait)
+                }
```

Comment:
Are `programs` and `channels` more important than people? If so I think they should go above them

Reply:
I have just attached them onto the bottom, even though it doesn't match the web. I will be introducing order customization in the future.